### ISSUE TITLE
Errors in "Array creation"

### ISSUE THREAD

**Issue by iouzzrj** [dotnet/docs#38364]
### Type of issue

Code doesn't work

### Description

[Enter feedback here]
You shouldn't use square brackets to initialize array like below:
int[] numbers = [10, 20, 30];

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/new-operator

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/operators/new-operator.md

### Document Version Independent Id

96df2e76-70b0-4873-4acc-d6bae51986b4

### Article author

BillWagner

### Metadata

* ID: c6bdb964-4c04-d174-e879-e3a2d176809b 
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**

**Comment by Rageking8** [comment]
That is a new feature released in C#12 [Collection Expressions](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-12#collection-expressions).

**Comment by BillWagner** [comment]
Hi @iouzzrj 

As @Rageking8 points out, this example is using the new collection expression syntax, added in C# 12. 

In order for developers to get more familiar with the syntax, we've updated our samples to use it. We did that at the time of the C# 12 release.

While it's true that it's not available in earlier versions, we find the syntax compelling, and we want to encourage its use.

I'll close this.
